### PR TITLE
CSS-147 Update SameSite in CSRF CS

### PR DESCRIPTION
**Pull request by ThunderSon**
Please make sure that for your contribution:

- [x] All the markdown files do not raise any validation policy violation, see policy [here](https://github.com/OWASP/CheatSheetSeries#editor--validation-policy).
- [x] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries#conversion-rules).
- [x] Any references to website have been formatted as [TEXT](URL)
- [x] You verified/tested the effectiveness of your contribution (e.g.: defensive code proposed is really an effective remediation? Please verify it works!).
- [x] The CI build of your PR pass, see the build status [here](https://travis-ci.org/OWASP/CheatSheetSeries/pull_requests).

This PR covers issue #147.

### CODE REVIEW

**Comment by righettod** [comment]
I will review it ASAP 😃

**Comment by ThunderSon** [comment]
I have a point that the guide needs a little bit of refactoring. It's huge, developers feel like it is daunting to go through it and it is huge! This is no longer a cheat sheet per say. Can you open an issue on this?

**Comment by mackowski** [comment]
> I have a point that the guide needs a little bit of refactoring. It's huge, developers feel like it is daunting to go through it and it is huge! This is no longer a cheat sheet per say. Can you open an issue on this?

Yes I will do it.

File: cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md [hunk 1]
```diff
@@ -213,24 +213,24 @@ Including the token in an encrypted cookie - often within the authentication coo
 
 ## Samesite Cookie Attribute
 
-SameSite is a cookie attribute (similar to HTTPOnly, Secure etc.) introduced by Google to mitigate CSRF attacks. It is defined in [this](https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-02#section-5.3.7) Internet Draft. This attribute helps in preventing the browser from sending cookies along with cross-site requests. Possible values for this attribute are lax or strict.
+SameSite is a cookie attribute (similar to HTTPOnly, Secure etc.) introduced by Google to mitigate CSRF attacks. It is defined in [rfc6265](https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-02#section-5.3.7). This attribute helps the browser decide when to send cookies along with cross-site requests. Possible values for this attribute are Lax or Strict.
```
[the remark this answers is not in the document]
Comment (ThunderSon) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: As None and the RFC6265bis are still in the testing phase, I'll open an issue to keep track of them, and once it is in production (out of Beta testing), it'll be added to the cheat sheet. 😄 Thank you for the heads up!

[the remark this answers is not in the document]
Comment (mikewest) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: My point was that `SameSite` isn't defined in RFC6265. :) It was originally defined in https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site which has been folded into https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis.

[the remark this answers is not in the document]
Comment (ThunderSon) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: Oh! I understood it in a different manner! Thank you for explaining what you meant 😄 
One additional question: How would `SameSite` handle `None` in the stable Chrome release, and not the Beta release? I am looking at it as such: If the developer took guidance and used `SameSite` as `None`, how would it behave currently, and how should it once released from Beta?

[the remark this answers is not in the document]
Comment (mikewest) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: In Chrome (and the spec before we added it explicitly), `SameSite=None` was treated the same as `SameSite=TotallyInvalidValue`: we drop the `SameSite` declaration entirely, and store the cookie as though it didn't have that declaration. This is how it works in every browser, except Safari 12, which treats `SameSite=TotallyInvalidValue` as `SameSite=Strict`, which is super-unfortunate. See the discussion in https://bugs.webkit.org/show_bug.cgi?id=198181. Hopefully they'll backport a fix, but who knows.

[the remark this answers is not in the document]
Comment (righettod) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: Perhaps my question will be stupid but I do not see in which case a usage of `SameSite=None` will be required/needed. Do you know one ?

**Comment by righettod** [comment]
As @mikewest and @mackowski have validate it, I merge it.
Thanks to you guys for the review and the discussion 😃

**Pull request merged**